tests/wcs1.py

```diff
@@ -298,17 +302,20 @@ class WCSTestCaseSDSS(unittest.TestCase):
     """A test case for WCS using a small (SDSS) image with a slightly weird WCS"""
 
     def setUp(self):
-        self.im = afwImage.DecoratedImageD(InputSmallImagePath)
-
-        self.wcs = afwImage.makeWcs(self.im.getMetadata())
-
-        if False:
-            ds9.mtv(self.im, wcs=self.wcs)
+        if afwdataDir is not None:
```

Comment:
I think this test should really be checking `InputSmallImagePath` is not None rather than `afwdataDir`.